src/backends/backend/examples/backend_utils.cc

```diff
@@ -35,10 +38,18 @@ std::string
 InstanceProperties::AsString() const
 {
   std::string str("id " + std::to_string(id_) + ", ");
-  if (kind_ == Kind::CPU) {
-    str += "CPU";
-  } else {
-    str += "GPU (" + std::to_string(device_id_) + ")";
+  switch (kind_) {
+    case Kind::CPU:
+      str += "CPU";
+      break;
+    case Kind::GPU:
+      str += "GPU (" + std::to_string(device_id_) + ")";
+      break;
+    case Kind::MODEL:
+      str += "MODEL";
+      break;
+    default:
```

Comment:
Don't add default, so that we get a compiler error if we add an enum and don't add it to this switch.